project-reports/2023/2023-Q4-Hyperledger-Aries.md

```diff
@@ -0,0 +1,171 @@
+    * [uniffi based mobile wrapper](https://wiki.hyperledger.org/pages/viewpage.action?pageId=98730187)
+
+**2023Q4 Planning**
+
+* Completion and stabilization of V2 issuance/presentation protocols
+* Stabilization of new reusable components created in 2023, such as did, did_document, did_peer, did_exchange and others. On the same note, general improvement of developer onboarding of these components, as they be adopted also outside of Aries context.
+* Completion of our two 2023 mentorship projects
+* Progress on work to migrate from credx to [anoncreds-rs] to enable implementation of non-indy anoncreds methods
+
+### [Aries Framework JavaScript]
+
+This quarter, [Aries Framework JavaScript] development was focused on two
+trends. On one side, making fixes and optimizations for 0.4.x release, following
+feedback from first adopters of it. Among those, we can highlight expanding
+support for the DIDComm Out-of-Band connection establishment flows and support
+for [Aries Askar] profiles in multi-tenancy scenarios. On the other side,
+preparing for the next major releases: 0.5.0 (expected for Q4 2023) and 0.6.0
+(expected for Q1 2024). Work towards those releases include supporting
+[SD-JWTs], the latest drafts of [OpenID4VCs] (issuance and presentation) and
+[DIDComm V2], are well underway.
+
+A topic of discussion in the [Aries Framework JavaScript] community is the
+possibility of the transfer of the [Aries Framework JavaScript] repository (and
+likely some related repositories) from the [Hyperledger Foundation] to the [Open
+Wallet Foundation] that has been [proposed by some Aries Framework JavaScript
+maintainers]. The issue is not yet finalized, and discussions are continuing.
+
+[DIDComm V2]: https://identity.foundation/didcomm-messaging/spec/
+
+### [Aries Cloud Agent Python]
+
+[Aries Cloud Agent Python] (ACA-Py) work in the quarter included the following highlights:
+
+* Making some foundational updates to the CI/CD pipeline (upgrading the minimum support Python version, switching to [Poetry] for package management, and [ruff] for linting).
+* Adding support for [SD-JWTs] format verifiable credential -- signing and verifying.
+* Work to enable the upcoming transitions from the use of unqualified DIDs, and to [DIDComm V2].
+* Adding support for the ledger-agnostic [Hyperledger AnonCreds].
+* Maintenance and performance optimizations.
+* Creation of the [Aries Cloud Agent Python plugins] repository -- an "app
+  store" for modular extensions to ACA-Py.
+
+[Poetry]: https://python-poetry.org/
+[ruff]: https://docs.astral.sh/ruff/
+
+### [Aries Framework Go]
+
+After an active previous quarter, activity slowed this quarter on [Aries
+Framework Go] for reasons not clear outside of the repository maintainers. We're
+trying get information from the maintainers on their plans for this component.
```

Comment:
```suggestion
trying to get information from the maintainers on their plans for this component.
```

Reply:
Updated the status of the Aries Framework Go after connecting with the primary maintainers.